templates/template.html

```diff
@@ -43,11 +43,11 @@
   <meta property="og:image" content="http://localhost:8080/dist/img/og-avatar.png">
   <meta property="og:image:width" content="400px">
   <meta property="og:image:height" content="400px">
-  <meta property="og:title" content="Warwick's Awesome Speedruns and Demos 2020">
+  <meta property="og:title" content="Warwick's Awesome Speedruns and Demos 2021">
```

Comment:
MD: This should be changed to reflect the event title proper

Reply:
Fixed in new commit